modules/console/pages/config/security/authentication.adoc

```diff
@@ -19,25 +19,31 @@
 include::shared:partial$enterprise-and-console.adoc[]
 ====
 
-This topic describes how to enable authentication in Redpanda Console and explains how it interacts with the Kafka API, Admin API, and Schema Registry API in Redpanda.
+This topic describes how to enable authentication in Redpanda Console and how it integrates with the Kafka, Admin, and Schema Registry APIs in Redpanda.
 
-Authentication in Redpanda Console lets users log in and optionally forward credentials to the connected cluster, ensuring API requests run under the user's identity. Redpanda Console supports two authentication methods:
+Authentication in Redpanda Console lets users log in and optionally forward their credentials to the connected cluster. This ensures API requests run under the user's identity.
 
-* **OIDC (OpenID Connect):**
-  Uses external identity providers, such as Google, to facilitate single sign-on (SSO).
-* **Basic authentication:**
-  Provides a traditional username and password login.
+Redpanda Console supports the following authentication methods:
+
+* **OIDC (OpenID Connect):** Integrates with external identity providers (IdPs) for single sign-on (SSO).
+* **Basic authentication:** Uses traditional username and password credentials.
 
 image::console:login.png[]
```

Comment:
Thoughts on figure name/numbers? Thinking that if/when we have to adhere to ADA 508 compliance/WCAG AA, we would have to add anyway. See https://www.wcag.com/authors/1-1-1-non-text-content/. Don't want this to hold up anything, but seeing this capture reminded me that we need to address this sooner than later.

Reply:
Ah--I see you have a figure callout down the page. Technically, isn't this considered a figure too? For example, if a blind user is reading the docs, I don't think they would have any way of know what this image file is showing. So I would argue it is a figure too.